packages/adminrouter/extra/src/test-harness/tests/test_agent.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+import grp
+
+import psutil
+import pytest
```

Comment:
Thanks, perhaps we should have the main dcos flake8 target run the adminrouter flake8 target, too? That way the 'code-quality' check will fail when we make this mistake again.